# Vedder VESC open source motor controller state (telemetry)

Header  header
float64 feedback
